dali/kernels/imgproc/jpeg/jpeg_distortion_gpu_test.cu

```diff
@@ -47,16 +50,18 @@ cv::Mat bgr2rgb(const cv::Mat& img) {
   return rgb2bgr(img);
 }
 
-template <typename GTestParams>
-class JpegDistortionTestGPU : public ::testing::Test {
-  using T = typename GTestParams::T;
-  static constexpr bool vert_subsample = GTestParams::vert_subsample;
-  static constexpr bool horz_subsample = GTestParams::horz_subsample;
-  static constexpr bool use_real_images = true;
-  static constexpr bool perf_run = false;
-  static constexpr bool dump_images = false;
+class JpegDistortionTestGPU : public ::testing::TestWithParam<std::tuple<bool, bool>> {
+  using T = uint8_t;
+  const bool horz_subsample;
+  const bool vert_subsample;
+  const bool use_real_images = true;
+  const bool perf_run = false;
+  const bool dump_images = false;
 
  public:
+  JpegDistortionTestGPU()
+      : horz_subsample(std::get<0>(GetParam())), vert_subsample(std::get<0>(GetParam())) {}
```

Comment:
Shouldn't it be ?
```suggestion
  JpegDistortionTestGPU()
      : horz_subsample(std::get<0>(GetParam())), vert_subsample(std::get<1>(GetParam())) {}
```
If it shouldn't I clearly didn't understood, what does `std::tuple<bool, bool>` denote, so maybe dropping line of documentation might come in handy

Reply:
True, that's a bug